build_debian.sh

```diff
@@ -204,6 +204,7 @@ sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y install      \
 sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install      \
     file                    \
     ifupdown2               \
+    ifmetric                \
```

Comment:
why do we need ifmetric?

Reply:
For setting the metric's for default and dhcp routes.